releases/2017_cycle_3.md

```diff
@@ -25,9 +25,12 @@ Scope:
     - More facility level metadata
     - Removal of `NAN`s from metadata
 - `Tlist` Shutter Protocol
```

Comment:
@sbillinge No, I didn't implement the new shutter protocol at the first place. I was worried about two points:
1. User  uses ``delay_time`` from ``tseries`` as the exact measure and it's down to ``sec`` (like the MED plan we supported before), so introducing uncontrollable shutter delays might be less preferable.
2. ``per_step`` doesn't apply to ``count`` which is the parent of ``tseries``.

After thinking a little bit, I guess 1 might not be a big issue as the user can override the behavior if he wishes. However, 2 might imply that we need to duplicate and maintain a considerable amount of ``bluesky`` source code   ([like here](https://github.com/NSLS-II/bluesky/blob/master/bluesky/plans.py#L2031)) to have our customized ``count`` plan.  That's is totally doable but slightly worries me.